<!DOCTYPE html>
<html>

  <head>
    <meta charset='utf-8'>
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <meta name="description" content="CopiaHT : Supplement page for poster on MEPS/HGC poster session Dec 2015">

    <link rel="stylesheet" type="text/css" media="screen" href="stylesheets/stylesheet.css">

    <title>CopiaHT</title>
  </head>

  <body>

    <!-- HEADER -->
    <div id="header_wrap" class="outer">
        <header class="inner">
          <a id="forkme_banner" href="https://github.com/Poaceae/CopiaHT">View on GitHub</a>

          <h1 id="project_title">CopiaHT</h1>
          <h2 id="project_tagline">Supplement page for poster on MEPS/HGC poster session Dec 2015</h2>

            <section id="downloads">
              <a class="zip_download_link" href="https://github.com/Poaceae/CopiaHT/zipball/master">Download this project as a .zip file</a>
              <a class="tar_download_link" href="https://github.com/Poaceae/CopiaHT/tarball/master">Download this project as a tar.gz file</a>
            </section>
        </header>
    </div>

    <!-- MAIN CONTENT -->
    <div id="main_content_wrap" class="outer">
      <section id="main_content" class="inner">
        <h2>
<a id="background--how-can-we-identify-horizontal-transfer-of-transposable-elements" class="anchor" href="#background--how-can-we-identify-horizontal-transfer-of-transposable-elements" aria-hidden="true"><span class="octicon octicon-link"></span></a>Background--how can we identify horizontal transfer of transposable elements?</h2>

<h2>
<a id="methods" class="anchor" href="#methods" aria-hidden="true"><span class="octicon octicon-link"></span></a>Methods</h2>

<h3>
<a id="sequence-alignment" class="anchor" href="#sequence-alignment" aria-hidden="true"><span class="octicon octicon-link"></span></a>Sequence alignment</h3>

<p>PtRLC_724_B homologous sequences in angiosperms were retrieved from blast results against NCBI nucleotide database (<a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">http://blast.ncbi.nlm.nih.gov/Blast.cgi</a>). For sequences in loblolly pine, sugar pine and Norway spruce genome, blast search was performed locally by BLAST+2.2.30. Sequences in Repbase were also retrieved through local blast. Representative sequences of each species (percent of identity of each other75% was set for local blast search.</p>

<p>PtRLC_724_B protein sequences were obtained through the tblastn results against the DNA sequences using Co4_Po_tr protein sequence in Repbase as the query.</p>

<p>Both DNA and protein sequences alignments were by MUSCLE  with default settings.</p>

<h3>
<a id="phylogenetic-tree-construction" class="anchor" href="#phylogenetic-tree-construction" aria-hidden="true"><span class="octicon octicon-link"></span></a>Phylogenetic tree construction</h3>

<p>Conserved sites for phylogenetic analyses were  selected based on the alignments by Gblocks0.91b under less stringent criteria “allow smaller final blocks” and “allow less strict flanking positions”. A total of 2995  nucleotide sites were kept after selection.</p>

<p>Substitution models for nucleotide sequences and amino acid sequences were selected using jModelTest2 and ProtTest 2.4, respectively.  jModelTest2 was ran on the CIPRES server (<a href="https://www.phylo.org/">https://www.phylo.org/</a> ).  Models were evaluated under all four criterions: AIC, BIC, AICc and LnL. The model listed as the best fit under most criterions was selected for phylogenetic tree construction. For PtRLC_724_B homologous sequences, the best nucleotide substitution model was GTR+G+F, and the best amino acid substitution model was JTT+G+F. </p>

<p>Maximum likelihood trees were built using the PhyML 3.0 program available on the ATGC bioinformatics platform (<a href="http://atgc.lirmm.fr/phyml/">http://atgc.lirmm.fr/phyml/</a> ), selecting options according to the best fitting substitution model. For trees based on DNA sequences, the number of substitution rate categories was set as 6 (to be consistent with that used in Bayesian tree). Both SPR and NNI were used for tree improvement. aLRT SH-like method was applied for branch support. One hundred  bootstrap iterations were also performed. </p>

<p>Bayesian trees were built with MrBayes3.2.6 on CIPRES (<a href="https://www.phylo.org/">https://www.phylo.org/</a> ). 5,000,000 generations MCMC were run and sampled every 100 generations; other options were selected according to the best fitting substitution model.</p>

<h2>
<a id="results" class="anchor" href="#results" aria-hidden="true"><span class="octicon octicon-link"></span></a>Results</h2>

<h3>
<a id="dna-based-phylogenies-of-ptrlc_724_b-homologous-copia-elements-in-angiosperms-and-conifers" class="anchor" href="#dna-based-phylogenies-of-ptrlc_724_b-homologous-copia-elements-in-angiosperms-and-conifers" aria-hidden="true"><span class="octicon octicon-link"></span></a>DNA-based phylogenies of PtRLC_724_B homologous copia elements in angiosperms and conifers</h3>

<h3>
<a id="protein-based-phylogenies-of-ptrlc_724_b-homologous-copia-elements-in-angiosperms-and-conifers" class="anchor" href="#protein-based-phylogenies-of-ptrlc_724_b-homologous-copia-elements-in-angiosperms-and-conifers" aria-hidden="true"><span class="octicon octicon-link"></span></a>Protein-based phylogenies of PtRLC_724_B homologous copia elements in angiosperms and conifers</h3>

<h2>
<a id="discussion" class="anchor" href="#discussion" aria-hidden="true"><span class="octicon octicon-link"></span></a>Discussion</h2>

<p>Three lines of evidence support a scenario whereby PtLRC_724_B Copia elements and their closely related families in conifers originated through a HTT
event from an ancestor of poplar to a common ancestor of pine trees and spruces.
First, both protein- and DNA-based phylogenies indicate that the Copia4 family in P. trichocarpa represents the closest lineage to the conifers PtLRC_724_B family (Fig. 3A-3B).  Because the Copia4+PtLRC_724_B monophyletic clade is embedded within Copia elements from other angiosperms, the transfer of PtLRC_724_B from conifers to poplar is not supported.
Second, the phylogeny of PtLRC_724_B and related angiosperm Copia families is incongruent with the species phylogeny (Fig. 3 C).
Third, PtLRC_724_B and other closely related conifer Copia families share a higher identity with the Copia4 family than either homologous genes or TEs found in loblolly pine and poplar.</p>

<h2>
<a id="ongoing-work" class="anchor" href="#ongoing-work" aria-hidden="true"><span class="octicon octicon-link"></span></a>Ongoing work</h2>

<p>Pine trees and spruces diverged around ~1325 million years ago; to determine the approximate timing of the PtLRC_724_B HTT event, we are performing an extensive survey of this family across non-Pinaceae gymnosperms.  We have designed degenerate primers based on the conserved region in the RT domains of conifer PtLRC_724_B families and the poplar Copia4 family to perform PCR assays on 32 non-Pinaceae gymnosperm.<br>
In addition, other possible cases of HTT between angiosperm and conifer genomes are being screened to determine the overall impact of this process in shaping the large genome of these gymnosperms</p>
      </section>
    </div>

    <!-- FOOTER  -->
    <div id="footer_wrap" class="outer">
      <footer class="inner">
        <p class="copyright">CopiaHT maintained by <a href="https://github.com/Poaceae">Poaceae</a></p>
        <p>Published with <a href="https://pages.github.com">GitHub Pages</a></p>
      </footer>
    </div>

              <script type="text/javascript">
            var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
            document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
          </script>
          <script type="text/javascript">
            try {
              var pageTracker = _gat._getTracker("UA-70948222-1");
            pageTracker._trackPageview();
            } catch(err) {}
          </script>


  </body>
</html>
